scripts/build-web-example.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+echo Upgrading examples RS version to $1
+
+var=`pwd`
+for d in packages/web/examples/*/ ; do
+    cd $var/$d
+    yarn add @appbaseio/reactivesearch@$1
```

Comment:
Why do we need to use yarn here, again it'll take a long time to execute this script?
Can't you just modify the package.json with the particular version?

Reply:
have removed `yarn` & now we are just changing package version.